WorkloadManagementSystem/Service/PilotsLoggingHandler.py

```diff
@@ -0,0 +1,58 @@
+""" PilotsLoggingHandler is the implementation of the PilotsLogging service
+
+    The following methods are available in the Service interface
+
+    addPilotsLogging()
+    getPilotsLogging
+    setPilotsUUIDtoIDMapping()
+    addPilotsUUID()
+    deletePilotsLogging()
+
+"""
+
+__RCSID__ = "$Id: $"
+
+from DIRAC.WorkloadManagementSystem.DB.PilotsLoggingDB import PilotsLoggingDB
+
+from DIRAC.Core.DISET.RequestHandler import RequestHandler
+from DIRAC import gLogger, S_OK, S_ERROR
+from DIRAC.Core.Utilities import Time
+
+class PilotsLoggingHandler( RequestHandler ):
+
+  @classmethod
+  def initializeHandler( cls, serviceInfo ):
+    return S_OK()
+
+  def initialize(self):
+    self.pilotsLogging = PilotsLoggingDB()
+
+  auth_addPilotsLogging = [ 'Operator' ]
```

Comment:
Why is it defined here ? It should rather go to the ConfigTemplate and to the CS finally